src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,17 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  setMovies: React.Dispatch<React.SetStateAction<Movie[]>>
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovies }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [spinner, setSpinner] = useState(false);
+
+  const onSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    getMovie(query)
+      .then(loadedMovie => {
+        if (loadedMovie === undefined || 'Error' in loadedMovie) {
+          setError(true);
+          setSpinner(false);
+
+          return;
+        }
+
+        let url = loadedMovie.Poster;
+
+        if (loadedMovie.Poster === '') {
+          url = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+        }
```

Comment:
you can do it like this to avoid reassignig value
```suggestion
        const url = loadedMovie.Poster 
          || 'https://via.placeholder.com/360x270.png?text=no%20preview';
```